tests/security/apparmor/aa_status.pm

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: apparmor tools testing - aa-status
```

Comment:
Can you provide a bit more detail what is tested?

Reply:
aa-status is a tool belong to apparmor-utils package. Which is used to check apparmor status as literal meaning.

Reply:
Yes, I know that. But can you state on the summary what this test is trying to cover

Reply:
Done. I have changed summary for a brief description for what it is cover